support/client/lib/vwf/model/mil-sym.js

```diff
@@ -55,7 +55,16 @@ define( [ "module",
                         "type": childType,
                         "name": childName 
                     };
-                }
+                },
+
+                // Public functions
+                "isUnitNode":                isUnitNode,
+                "isMissionGfxNode":          isMissionGfxNode,
+                "isModifierNode":            isModifierNode,
+                "getModifierActualName":     getModifierActualName,
+                "convertModifierValue":      convertModifierValue,
+                "computeRelativeControlPts": computeRelativeControlPts,
+                "getMilSymControlPts":       getMilSymControlPts
```

Comment:
For those functions that are used in only the model driver (all except for `getModifierActualName` and `convertModifierValue`), let's keep them private.  That way we have more flexibility to change them without having to worry about supporting a backward compatible API.